beachball.config.dev.js

```diff
@@ -7,7 +7,7 @@ const base = require("./beachball.config.js");
 /** @type {import("beachball").BeachballConfig } */
 module.exports = {
   ...base,
-  scope: [],
+  scope: ["packages/itwin/map-layers"],
```

Comment:
@grigasp @saskliutas is there anything else that needs to be done for the pipeline to later publish/bump map-layers to `6.0.0-dev.0`?

Reply:
I think you also need to set version to `6.0.0-dev` in `package.json` manually. You can test it out locally by running `pnpm beachball bump --config beachball.config.dev.js --keep-change-files` at the root of this repo.

Don't forget to tick `Publish pre-release versions` checkbox before starting the pipeline.

Reply:
alright, just did that, and tested things out locally, the command will bump the ver to `6.0.0-dev.1`, so expecting the pipeline to do that

Reply:
You can set it to `6.0.0-dev` if you want first version to be `6.0.0-dev.0` instead of `6.0.0-dev.1`

Reply:
ah i see, thank you, just changed it